core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -1324,6 +1324,16 @@ public void testParseSchemaIdentifierFields() throws Exception {
     Assert.assertEquals(Sets.newHashSet(1, 2), parsed.schemasById().get(1).identifierFieldIds());
   }
 
+  @Test
+  public void testParseMinimal() throws Exception {
+    String data = readTableMetadataInputFile("TableMetadataV2ValidMinimal.json");
+    TableMetadata parsed = TableMetadataParser.fromJson(data);
+    Assert.assertEquals(Lists.newArrayList(), parsed.snapshots());
```

Comment:
nit: Iceberg is currently migrating away from JUnit4, meaning that it later adds additional work to convert these JUnit4-style assertions. It would be great to update these to AssertJ.

```
    assertThat(parsed.snapshots()).isEmpty();
    assertThat(parsed.snapshotLog()).isEmpty();
    assertThat(parsed.properties()).isEmpty();
    assertThat(parsed.previousFiles()).isEmpty();
```

Reply:
🤦 Thanks! Just copied this from above :)